article_parser.py

```diff
@@ -42,39 +42,41 @@ def character_recognition(self, text):
         text = phrase2.sub('.', text)
         text = phrase3.sub('', text)
         text = text.split(".") #распарсили текст
-        self.useful_operations(text)
+        self.pre_operations(text)
         self.critere_parser()
         return self.probable_names
 
     def critere_parser(self):
-        for index in range(len(self.sentences_list)): #имеет смысл иметь зип заглавных всех, заглавных первых (так будет быстрее)
+        def estimator(cur_word, num_word):
```

Comment:
Полгода назад ><
Океей, намек понят, все сделаю хд
--
Envoyé de l'application Mail.Ru pour Android vendredi, 09 septembre 2016, 04:26PM +03:00 de Cadovvl < notifications@github.com> :

>In  article_parser.py :
>>          self.critere_parser()
>          return self.probable_names
>  
>      def critere_parser(self):
> -        for index in range(len(self.sentences_list)): #имеет смысл иметь зип заглавных всех, заглавных первых (так будет быстрее)
> +        def estimator(cur_word, num_word):
>Хз. Если ты уже не помнишь этот код, то лучше не стоит. Полгода назад стоило бы...
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .